frontend/src/components/AdequacyCensusCharts/AdequacyCensusCharts.tsx

```diff
@@ -52,22 +53,36 @@ export let AdequacyCensusCharts = withStore('adequacies', 'method')<Props>(({ se
       </tr>
       {
         censusGroups.map(censusGroup =>
-          adequacyRowByCensusGroup(censusGroup, populationByAdequacyByGroup[censusGroup])
+          adequacyRowByCensusGroup(censusGroup, populationByAdequacyByGroup[censusGroup], format)
         )
       }
     </StatsBox>
   </div>
 })
 
-function adequacyRowByCensusGroup(censusGroup: string, populationByAdequacy: PopulationByAdequacy) {
+function adequacyRowByCensusGroup(censusGroup: string, populationByAdequacy: PopulationByAdequacy, format: Format) {
   let totalPopulation = populationByAdequacy.reduce((a: number, b: number) => a + b)
+  let adequacyColumnIndices = [0, 1, 2, 3]
+  if (format === 'Percentage') {
+    return (
+      <tr>
+        <td>{censusGroup}</td>
+        {
+          adequacyColumnIndices.map(i =>
+            (<td>{formatPercentage(100 * populationByAdequacy[i] / totalPopulation)}</td>)
+          )
+        }
+      </tr>
+    )
+  }
   return (
     <tr>
       <td>{censusGroup}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[0] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[1] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[2] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[3] / totalPopulation)}</td>
+      {
+        adequacyColumnIndices.map(i =>
```

Comment:
i think you can map the items in the list directly. 
populationByAdequacy.map(_ => <td>formatNumber(_)</td> or do

Reply:
Actually I cannot because i do not want to map the last one!